packages/fluent-theme/src/components/activity/private/useActivityAuthor.ts

```diff
@@ -0,0 +1,16 @@
+import { useMemo } from 'react';
+import { getOrgSchemaMessage, type WebChatActivity } from 'botframework-webchat-core';
+
+export default function useActivityAuthor(activity?: WebChatActivity | undefined) {
+  return useMemo(() => {
+    const entity = getOrgSchemaMessage(activity?.entities || []);
+    return typeof entity?.author === 'string'
+      ? {
+          '@type': 'Person',
+          description: undefined,
+          image: undefined,
```

Comment:
Do we need to specify them as `undefined`?

Reply:
Needed initially to convey typescript that this is `Person` compatible